### DISCUSSION TITLE
SliderInput

### DISCUSSION THREAD

**Discussion by yaurita** [teamalgoritma/community#402] [Q&A] [answered]
Hai hai, mau nanya untuk tugas Capstone DV.
Untuk input, ketika saya buat 1 slider grafiknya muncul. Namun ketika saya buat 2 slider, grafiknya tidak muncul.
Ada salah dimanakah? Mohon reviewnya. Makasih :))

--Server--
shinyServer(function(input, output) {
  
  output$plot_1 <- renderPlotly({
    case1 <- loan_clean %>% 
      filter(Gender %in% input$check_gender, 
             Country == input$choice_country,
             LoanDuration > input$LoanDuration[1],
             LoanDuration < input$LoanDuration[2],
             Amount > input$Amount[1],
             Amount < input$Amount[2]) %>% 
      group_by(year_month) %>%   
      summarise(count = n()) %>% 
      mutate(label = glue("Jumlah Debitur: {count}"))

--UI--
# ROW 2
fluidRow(
# BOX 1 : checkBoxGroup Input
box(width = 4,
height = 100,
checkboxGroupInput(inputId = "check_gender",
                                       label = "Choose Gender :",
                                       choices = unique(loan_clean$Gender),
                                       selected = c('Male','Female'),
                                       inline = T
                                       
                    )
                ),

# BOX 2 : Slider Input
box(width=4,
height = 100,
sliderInput("LoanDuration", 
                              label = "Choose LoanDuration:",
                              min = min(loan_clean$LoanDuration), 
                              max = max(loan_clean$LoanDuration),
                              value = c(0,60),
                              step = 12)    
                  ),

# BOX 3 : Slider Input
box(width=4,
height = 100,
sliderInput("Amount", 
                                label = "Choose Amount:",
                                min = min(loan_clean$Amount), 
                                max = max(loan_clean$Amount),
                                value = c(0,11000),
                                step = 3000)
                
                    )
              ),

<img width="905" alt="image" src="https://user-images.githubusercontent.com/116186752/196730045-d893c028-4151-45e9-8709-379c0eca2b75.png">

**Comment by NabiilahArdini** [comment, marked as answer]
Dear @yaurita ,

Dari code yang Anda lampirkan tidak ada kesalahan pada koneksi antara inputId dan versi render nya di server. Apakah Anda bisa melampirkan code full dari `output$plot_1` ?

Selain itu, apakah Anda bisa menampilkan apakah ada warning/error message yang lebih detail dari console Anda?

Terima kasih.
